test/functional/connection/connection.ts

```diff
@@ -381,33 +387,89 @@ describe("Connection", () => {
+                    enabledDrivers: ["postgres"],
+                    entities: [CommentV1, GuestV1],
+                    schemaName: "another-schema",
+                    dropSchemaOnConnection: true
+                })
+            ]);
+            connections = [connection1, connection2];
+        });
+        after(() => closeTestingConnections(connections));
+        it("should not interfere with each other", async () => {
+            await Promise.all(connections.map(c => c.syncSchema()));
+            await closeTestingConnections(connections);
+            const [[connection1], [connection2]] = await Promise.all([
+                createTestingConnections({ 
+                    enabledDrivers: ["postgres"],
+                    entities: [CommentV2, GuestV2],
+                    schemaName: "test-schema",
+                    dropSchemaOnConnection: false,
+                    schemaCreate: true
+                }),
+                createTestingConnections({ 
+                    enabledDrivers: ["postgres"],
+                    entities: [CommentV2, GuestV2],
+                    schemaName: "another-schema",
+                    dropSchemaOnConnection: false,
+                    schemaCreate: true
+                })
+            ]);
+            connections = [connection1, connection2];
+        });
+    });
+
+    describe.only("Can change postgres default schema name", () => {
+        let connections: Connection[];
+        beforeEach(async () => {
+            const [[connection1], [connection2]] = await Promise.all([
+                createTestingConnections({ 
+                    enabledDrivers: ["postgres"],
+                    entities: [CommentV1, GuestV1],
+                    schemaName: "test-schema",
+                    dropSchemaOnConnection: true
+                }),
+                createTestingConnections({ 
+                    enabledDrivers: ["postgres"],
+                    entities: [CommentV1, GuestV1],
+                    schemaName: "another-schema",
+                    dropSchemaOnConnection: true
+                })
+            ]);
+            connections = [connection1, connection2];
         });
         afterEach(() => closeTestingConnections(connections));        
+
         it("schema name can be set", () => {
             return Promise.all(connections.map(async connection => {
                 await connection.syncSchema(true);
+                const schemaName = (connection.driver as PostgresDriver).schemaName;
+                const comment = new CommentV1();
+                comment.title = "Change SchemaName";
+                comment.context = `To ${schemaName}`;
 
-                const post = new Post();
-                post.title = "ChangeSchemaName";
-
-                const PostRepo = connection.getRepository(Post);
-                await PostRepo.persist(post);
+                const CommentRepo = connection.getRepository(CommentV1);
```

Comment:
please name repositories as standard - camelCase and simple like: `commentRepository` or simply `repository`